src/app/modules/payments/stripe.utils.ts

```diff
@@ -435,11 +476,25 @@ export const convertToProofOfPaymentFormat = (
       '<td class="st-Spacer st-Spacer--gutter" width="48"></td>',
     )
 
+  // Check if payment by products (instead of variable / fixed payments) is used. If yes, modify proof of payment
+  const paymentByProductsEdits =
+    products.length === 0
```

Comment:
/nit Since `products.length === 0` implies that it is not a payment by products, I think it might be useful to put this implicit assumption down

Reply:
[d00030f](https://github.com/opengovsg/FormSG/pull/6574/commits/d00030faa17fcb6fb88f28ec126998d633569e3d)